acceptance/suites/tests/code_commands/code_scripts.rb

```diff
@@ -0,0 +1,181 @@
+require 'json'
+
+confine :except, :type => 'pe'
+
+test_name 'SERVER-1118: Validate code-id-command feature in FOSS'
+
+git_repo='/git/puppetcode' #AKA "Control Repo" &| "Git Server" &| "origin"
+git_local_repo='/tmp/git'
```

Comment:
Spaces around operators? Not real crazy about the comment at the end of the line, either.